virtwho/config.py

```diff
@@ -953,8 +962,8 @@ def get_defaults(cls):
 
     @classmethod
     def class_for_type(cls, virt_type):
-        clazz = ConfigSection
-        # TODO: Update to use actual subclasses as the are created
+        clazz = cls
+        # TODO: Update to use actual subclasses as they are created
```

Comment:
TODO: remove this TODO :-), because we will not to implement it .In theory `type` would be needed during initialization of object and `__new__(cls, *args, **kwargs` could return desired subsclass.